docs/developer-docs/latest/developer-resources/database-apis-reference/rest-api.md

```diff
@@ -158,18 +158,7 @@ Returns entries matching the query filters (see [parameters](#api-parameters) do
       "id": 1,
       "attributes": {
         "title": "Restaurant A",
-        "description": "Restaurant A's description",
-        "categories": [
-          {
-            "id": 1,
-            "attributes": {
-              "name": "My category"
-            },
-            "meta": {
-              "availableLocales": []
-            }
-          }
-        ]
+        "description": "Restaurant A's description"
```

Comment:
Your suggested change is because we're not using `?populate=categories`, here, right?

Thank you, good catch!

Reply:
Yes